test/rijksmus-api/client_test.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../../lib/rijksmus-api'
+require 'test_helper'
+
+describe Rijksmus::API::Client, :vcr do
+  extend Minitest::Spec::DSL
+
+  let(:search_term){"Karlsruhe"}
```

Comment:
it's totally code style, but with spaces it's better readable.`let(:search_term) { "Karlsruhe" }` and closer to coding conventions. In this example it doesn't matter, just as a general advise.
For these kind of things it's worth checking out rubocop that can actually fix these things automatically.